RetrospectiveExtension.Backend/deploy/env_setup.sh

```diff
@@ -62,7 +62,22 @@
     az appservice plan create \
         --name "plan-${resource_name_suffix}" \
         --resource-group "$resource_group"
-    
+
+    # https://docs.microsoft.com/en-us/cli/azure/monitor/app-insights/component?view=azure-cli-latest#az-monitor-app-insights-component-create
+    # Create Application Insights resource
+    echo "### Creating Application Insights resource"
```

Comment:
Add Trailing #### so that it looks consistent with the rest of the echo statements in the script  for example
`echo "#### Publishing backend project ####"`